log.go

```diff
@@ -36,18 +36,30 @@ type raftLog struct {
 	// been instructed to apply to its state machine. Some of these
 	// entries may be in the process of applying and have not yet
 	// reached applied.
+	// Use: The field is incremented when accepting a Ready struct.
 	// Invariant: applied <= applying && applying <= committed
 	applying uint64
 	// applied is the highest log position that the application has
 	// successfully applied to its state machine.
+	// Use: The field is incremented when advancing after the committed
+	// entries in a Ready struct have been applied (either synchronously
+	// or asynchronously).
 	// Invariant: applied <= committed
 	applied uint64
 
 	logger Logger
 
-	// maxNextCommittedEntsSize is the maximum number aggregate byte size of the
-	// messages returned from calls to nextCommittedEnts.
-	maxNextCommittedEntsSize uint64
+	// maxApplyingEntsSize is the maximum outstanding byte size of the messages
```

Comment:
The comment read slightly ambiguous to me. I thought it was tracking the maximum apply batch we had seen during the lifetime of the instance, and why we would need that had me confused.

I then realized this is simply the limit. Maybe clarify this for dummies like me

```suggestion
// maxApplyingEntsSize limits the outstanding byte size of the messages
```

Reply:
Done.